Entity-ngsiv2-schema.json

```diff
@@ -0,0 +1,185 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/Entity-ngsiv2-schema.json",
+  "title": "NGSIv2 Entity",
+  "description": "NGSIv2 Entity",
+  "definitions": {
+    "EntityIdentifier": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 256,
+      "pattern": "^[\\w\\-\\.\\{\\}\\$\\+\\*\\[\\]`|~^@!,:\\\\]+$"
+    },
+    "Name": {
+      "type": "string",
+      "pattern": "^(\\d|[a-zA-Z]|_)+$",
+      "minLength": 1,
+      "description": "NGSIv2 Name"
+    },
+    "Value": {
+      "type": [
+        "number",
+        "boolean",
+        "array",
+        "object",
+        "null",
+        "string"
+      ]
+    },
+    "TypeValue": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/Name"
+            },
+            {
+              "not": {
+                "const": "DateTime"
+              }
+            }
+          ]
+        },
+        "value": {
+          "$ref": "#/definitions/Value"
+        }
+      },
+      "required": [
+        "value"
+      ]
+    },
+    "DateTimeValue": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "DateTimeTypeValue": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "const": "DateTime"
+        },
+        "value": {
+          "$ref": "#/definitions/DateTimeValue"
+        }
+      },
+      "required": [
+        "type",
+        "value"
+      ]
+    },
+    "GeoTypeValue": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "type": "string",
+          "const": "geo:json"
+        },
+        "value": {
+          "$ref": "http://json-schema.org/geojson/geometry.json#"
+        }
+      },
+      "required": [
+        "type",
+        "value"
+      ]
+    },
+    "Metadata": {
+      "type": "object",
+      "additionalProperties": {
+        "anyOf": [
+          {
+            "$ref": "#/definitions/TypeValue"
+          },
+          {
+            "$ref": "#/definitions/DateTimeTypeValue"
+          }
+        ]
+      }
+    },
+    "dateCreated": {
+      "$ref": "#/definitions/DateTimeTypeValue"
+    },
+    "dateModified": {
+      "$ref": "#/definitions/DateTimeTypeValue"
+    },
+    "Attribute": {
+      "allOf": [
+        {
+          "oneOf": [
+            {
+              "$ref": "#/definitions/DateTimeTypeValue"
+            },
+            {
+              "$ref": "#/definitions/GeoTypeValue"
+            },
+            {
+              "$ref": "#/definitions/TypeValue"
+            }
+          ]
+        },
+        {
+          "type": "object",
+          "properties": {
+            "metadata": {
+              "$ref": "#/definitions/Metadata"
+            }
+          }
+        }
+      ]
+    },
+    "EntityFragment": {
```

Comment:
What about the keyValues an valude entity representations described in the "Simplified Entity Representation" section of the spec? Does this schema cover them?

If not, also fine with it (the normalized format is the most complete one, so it makes sense we focus on it) but it should be taken into account and documented as limitation is some way.

Reply:
as you say this is only covering the normalized format

Reply:
Thanks for the clarification. Clarification about it should be included in the PR.

Reply:
Maybe specifying it in the root "description" field and in the filename of the schema would suffice.

Reply:
fixed https://github.com/Fiware/dataModels/pull/373/commits/4599e4602ea40bb543bff6338fed72ee99806b60